src/definitions/elements/icon.less

```diff
@@ -508,7 +508,7 @@ i.icons .icon {
   top: 50%;
   left: 50%;
   transform: translateX(-50%) translateY(-50%);
-  margin: 0em;
+  margin: 0;
   margin: 0;
```

Comment:
margin is defined twice

Reply:
This was before this PR, https://github.com/fomantic/Fomantic-UI/blob/991a19549324a11ffc2601bf56b51ce30b46d571/src/definitions/elements/icon.less#L511 I will fix anyway 😉